Kerberos package imports break build after move to v5/ subdirectory

### Summary
After commit 3d7eaf1 (\"Moved kerberos to v5\") moved `network/kerberos/*` to `network/kerberos/v5/`, none of the import paths inside the moved files or their external consumers were updated. `go build ./...` fails at the repository root with three \"no required module provides package\" errors.

### Location
- **File(s):**
  - `network/kerberos/v5/client.go` (lines 11–12)
  - `network/kerberos/v5/asreproast.go` (line 10)
  - `network/kerberos/v5/crypto/crypto.go` (line 14, plus doc comment on line 5)
  - `network/kerberos/v5/crypto/crypto_test.go` (line 9)
  - `network/ldap/session.go` (line 9)

### Category
`functional`

### Severity
`critical`

The repository does not compile on `main`. Any consumer doing `go build ./...`, `go test ./...`, or depending on this module fails immediately.

### Reproduction / Evidence
Reproducible at runtime:

```
$ go build ./...
network/kerberos/v5/client.go:11:2: no required module provides package github.com/TheManticoreProject/Manticore/network/kerberos/crypto; to add it:
        go get github.com/TheManticoreProject/Manticore/network/kerberos/crypto
network/kerberos/v5/asreproast.go:10:2: no required module provides package github.com/TheManticoreProject/Manticore/network/kerberos/messages; to add it:
        go get github.com/TheManticoreProject/Manticore/network/kerberos/messages
network/ldap/session.go:9:2: no required module provides package github.com/TheManticoreProject/Manticore/network/kerberos; to add it:
        go get github.com/TheManticoreProject/Manticore/network/kerberos
```

Verified by code analysis:

- `client.go:11` imports `github.com/TheManticoreProject/Manticore/network/kerberos/crypto`, but after the move the actual package lives at `network/kerberos/v5/crypto`.
- `client.go:12` and `asreproast.go:10` import `.../network/kerberos/messages`; the package now lives at `.../network/kerberos/v5/messages`.
- `crypto/crypto.go:14` and `crypto/crypto_test.go:9` have the same problem (import `.../network/kerberos/messages`).
- `network/ldap/session.go:9` imports the top-level `.../network/kerberos` package; after the move, the public entrypoint lives at `.../network/kerberos/v5`.

### Expected Behavior
`go build ./...` succeeds on a freshly cloned `main`.

### Actual Behavior
Compilation fails with unresolved imports in three locations.

### Root Cause
Commit 3d7eaf1 (\"Moved kerberos to v5\") moved the files under `network/kerberos/` into `network/kerberos/v5/` with `git mv`, but no import paths in the moved files — or in the out-of-tree consumer `network/ldap/session.go` — were updated to reflect the new path.

--- a/network/kerberos/v5/asreproast.go
+++ b/network/kerberos/v5/asreproast.go
@@ -7,7 +7,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/TheManticoreProject/Manticore/network/kerberos/messages"
+	"github.com/TheManticoreProject/Manticore/network/kerberos/v5/messages"
 )
 
 // ASREPRoastResult contains the raw fields extracted from an AS-REP response for an

--- a/network/kerberos/v5/client.go
+++ b/network/kerberos/v5/client.go
@@ -8,8 +8,8 @@ import (
 	"strings"
 	"time"
 
-	kerbcrypto "github.com/TheManticoreProject/Manticore/network/kerberos/crypto"
-	"github.com/TheManticoreProject/Manticore/network/kerberos/messages"
+	kerbcrypto "github.com/TheManticoreProject/Manticore/network/kerberos/v5/crypto"
+	"github.com/TheManticoreProject/Manticore/network/kerberos/v5/messages"
 )
 
 // KerberosClient manages Kerberos authentication against an Active Directory KDC.

--- a/network/kerberos/v5/crypto/crypto.go
+++ b/network/kerberos/v5/crypto/crypto.go
@@ -2,7 +2,7 @@
 // string-to-key derivation, encryption, and decryption for RC4-HMAC and
 // AES-CTS-HMAC-SHA1-96 encryption types.
 //
-// Import path: github.com/TheManticoreProject/Manticore/network/kerberos/crypto
+// Import path: github.com/TheManticoreProject/Manticore/network/kerberos/v5/crypto
 package kerbcrypto
 
 import (
@@ -11,7 +11,7 @@ import (
 	"fmt"
 	"io"
 
-	"github.com/TheManticoreProject/Manticore/network/kerberos/messages"
+	"github.com/TheManticoreProject/Manticore/network/kerberos/v5/messages"
 )
 
 // Key usage constants per RFC 4120 Section 7.5.1.

--- a/network/kerberos/v5/crypto/crypto_test.go
+++ b/network/kerberos/v5/crypto/crypto_test.go
@@ -6,7 +6,7 @@ import (
 	"testing"
 
 	"github.com/TheManticoreProject/Manticore/crypto/nt"
-	"github.com/TheManticoreProject/Manticore/network/kerberos/messages"
+	"github.com/TheManticoreProject/Manticore/network/kerberos/v5/messages"
 )
 
 // ---------------------------------------------------------------------------

--- a/network/ldap/session.go
+++ b/network/ldap/session.go
@@ -6,7 +6,7 @@ import (
 
 	"github.com/TheManticoreProject/Manticore/windows/credentials"
 
-	"github.com/TheManticoreProject/Manticore/network/kerberos"
+	kerberos "github.com/TheManticoreProject/Manticore/network/kerberos/v5"
 	"github.com/go-ldap/ldap/v3"
 	"github.com/go-ldap/ldap/v3/gssapi"
 	krb5client "github.com/jcmturner/gokrb5/v8/client"

